src/Creditor.sol

```diff
@@ -57,25 +57,26 @@ abstract contract Creditor {
 
     /**
      * @notice Sets the validity of Account version to valid.
-     * @param accountVersion The version current version of the Account.
-     * @param valid The validity of the respective accountVersion.
+     * @param accountVersion The current version of the Account.
+     * @param isValid Will be "true" if respective Account version is valid, "false" if not.
      */
-    function _setAccountVersion(uint256 accountVersion, bool valid) internal {
-        isValidVersion[accountVersion] = valid;
+    function _setAccountVersion(uint256 accountVersion, bool isValid) internal {
+        isValidVersion[accountVersion] = isValid;
 
-        emit AccountVersionSet(accountVersion, valid);
+        emit ValidAccountVersionsUpdated(accountVersion, isValid);
     }
 
     /**
-     * @notice Checks if Account fulfills all requirements and returns application settings.
-     * @param accountVersion The current version of the Account.
+     * @notice Checks if Account fulfills all requirements and returns creditor settings.
+     * @param accountVersion The version of the Arcadia Account.
      * @return success Bool indicating if all requirements are met.
-     * @return baseCurrency The base currency of the application.
-     * @return liquidator The liquidator of the application.
+     * @return baseCurrency The base currency of the creditor.
+     * @return liquidator The liquidator of the creditor.
      * @return fixedLiquidationCost Estimated fixed costs (independent of size of debt) to liquidate a position.
      */
     function openMarginAccount(uint256 accountVersion)
         external
+        view
```

Comment:
Not sure if we have to force it to be a view function in the abstract contract, can imagine in some cases that you want to 'register' the Accounts for some reason on the Creditor.